openpower/package/zaius-xml/zaius.mk

```diff
@@ -0,0 +1,89 @@
+################################################################################
+#
+# zaius_xml
+#
+################################################################################
+
+ZAIUS_XML_VERSION ?= db53313f3873fc0a822121c0d95f257604387a76
+ZAIUS_XML_SITE ?= $(call github,rlippert,zaius-xml,$(ZAIUS_XML_VERSION))
```

Comment:
Yea that would be great as it was always the intent.  I think you can just clone my repository under the open-power github organization.